spec/requests/user_request_spec.rb

```diff
@@ -27,6 +27,7 @@
       let(:mhv_user) { build(:user, :mhv, :no_mpi_profile) }
 
       it 'GET /v0/user - returns proper json' do
+        create(:mhv_user_verification, mhv_uuid: mhv_user.mhv_correlation_id)
```

Comment:
had to add these, on order to prevent a need to add a check for `user_account` in `veteran_onboarding.rb`